skimage/measure/tests/test_colocalization.py

```diff
@@ -51,13 +51,11 @@ def test_pcc():
     assert pearson_corr_coeff(img1, img1) == (1.0, 0.0)
 
     img2 = np.where(img1 <= 2, 0, img1)
-    assert pearson_corr_coeff(img1, img2) \
-           == (0.944911182523068, 3.5667540654536515e-08)
+    np.testing.assert_almost_equal(pearson_corr_coeff(img1, img2), (0.944911182523068, 3.5667540654536515e-08))
 
     # change background of roi and see if values are same
     roi = np.where(img1 <= 2, 0, 1)
-    assert pearson_corr_coeff(img1, img1, roi) \
-           == pearson_corr_coeff(img1, img2, roi)
+    np.testing.assert_almost_equal(pearson_corr_coeff(img1, img1, roi), pearson_corr_coeff(img1, img2, roi))
```

Comment:
I am not sure this is necessary, since the results are both being generated by `pearson_corr_coeff` and not comparing againsts hard-coded values.